sample-tracking-frontend/src/components/searchform.js

```diff
@@ -1,4 +1,4 @@
-
+/* eslint-disable */ 
```

Comment:
need to remove this line once other changes are ready. This line does not appear necessary for non-local deploys.